Source/bmalloc/bmalloc/TZoneHeapManager.h

```diff
@@ -147,18 +150,33 @@ class TZoneHeapManager {
 
     BINLINE static TZoneHeapManager& singleton()
     {
-        if (!theTZoneHeapManager)
+        if (!theTZoneHeapManager) {
             ensureSingleton();
+            theTZoneHeapManager->init();
+        }
         BASSERT(theTZoneHeapManager);
         return *theTZoneHeapManager;
     }
 
+    BINLINE static void initWithBucketSettings(unsigned smallSizeCount, unsigned largeSizeCount = 0, unsigned smallSizeLimit = 0)
```

Comment:
This function is not used, right?

Reply:
It isn't used.  I'll use it in a subsequent patch.